lib/active_pivot/api/project.rb

```diff
@@ -2,7 +2,7 @@ module ActivePivot
   module Api
     class Project < OpenStruct
       def self.all
-        Api::PaginatedCollection.new("/services/v5/projects.json").all.map do |remote_project|
+        Api::PaginatedCollection.new("/services/v5/projects.json?fields=:default,current_velocity,current_volatility").all.map do |remote_project|
```

Comment:
Colon needs to be there according to API example:
`GET /services/v5/projects/1/epics/5?fields=:default,comment_ids`

I agree about the clunky URL. Unfortunate that Pivotal leaves these important attributes out by default (initial_velocity is default but current_velocity isn't? Why, Pivotal?)